pages/api/studies/[adminName].js

```diff
@@ -0,0 +1,15 @@
+import dbConnect from '@lib/db-connect'
+import Study from '@models/study'
+
+export default async function handler(req, res) {

```

Comment:
For sure. I'll create a user story to research and design out a plan for auth, probably using [this](https://next-auth.js.org/). Along these lines I was also thinking the app will probably be invite only which would help secure it somewhat and also help keep the user base under control (so we can use free tier hosting solutions).

Reply:
Sounds good to me. Instead of making the app invite only on the App Store we could just make it unusable without a study key / admin created account so students don't run into installation issues.